Sources/ReleaseServer/ReleaseServer.WebApi/Models/DeploymentMetaInfoModel.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace ReleaseServer.WebApi.Config
 {
-    public class DeploymentMetaInfo
+    public class DeploymentMetaInfoModel
```

Comment:
Why did you add the suffix "Model". IMHO this is only necessary/useful, if there also is a class `DeploymentMetaInfo` and this is the corresponding model for that.

Reply:
I removed the "Model" suffix, see https://github.com/Traeger-GmbH/release-server/commit/014b059c85cec990a6c6c065dd406057b121f649